lib/workers/repository/updates/generate.js

```diff
@@ -254,6 +254,8 @@ function generateBranchConfig(branchUpgrades) {
   }
   // Now assign first upgrade's config as branch config
   config = { ...config, ...config.upgrades[0], releaseTimestamp };
+  // check at line #217 sets releaseTimestamp to the latest upgrade ts
```

Comment:
Careful with uses `#` as that leads to linking to issues/PRs. Also, lines can get changed at any time so it's not the best reference.

Anyway, it seems like we already have calculated the most recent timestamp, so I think we should either reuse that existing field or rename it if we think its current name is unclear.

Reply:
better use the same property name because it may be used in some other places